src/main/kotlin/com/ajaxproject/telegrambot/bot/BankInfoBot.kt

```diff
@@ -1,36 +1,49 @@
 package com.ajaxproject.telegrambot.bot
 
-import com.ajaxproject.telegrambot.bot.commands.AbstractCommand
+import com.ajaxproject.telegrambot.bot.model.UserRequest
+import com.ajaxproject.telegrambot.bot.service.UserSessionService
 import org.slf4j.LoggerFactory
-import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Component
 import org.telegram.telegrambots.bots.TelegramLongPollingBot
 import org.telegram.telegrambots.meta.api.objects.Update
-
 @Component
 class BankInfoBot(
-    @Qualifier("commandsMap")
-    private val commandsMapping: Map<String, AbstractCommand>,
+    val dispatcher: Dispatcher,
+    val userSessionService: UserSessionService,
 ) : TelegramLongPollingBot() {
-
-    @Value("\${credentials.botUsername}")
+    @Value("\${bot.username}")
     private val botUsername: String = ""
 
-    @Value("\${credentials.botToken}")
+    @Value("\${bot.token}")
```

Comment:
The same advice as in https://github.com/mykolaKhlopyk/EnglishStudyTelegramBot/pull/1#discussion_r1323663378